### PR TITLE
Revised CodeCoverage parameter description

### PR DESCRIPTION
**Pull request by juneb**

### CODE REVIEW

File: Pester.psm1 [hunk 1]
```diff
@@ -220,14 +220,42 @@ parameter.
...
+One of the following: Function or StartLine/EndLine
+	
+-- Function (F) <string>: Enter the function name. Wildcard characters are supported, 
+   but only one string is permitted.
+	
+-or-
+	
+-- StartLine (S): Performs code coverage analysis beginning with the specified line. Default
+   is line 1.
+-- EndLine (S): Performs code coverage analysis ending with the specified line. Default is
```
Comment (dlwyatt) [inline Pester.psm1]: This should be EndLine (E)

**Comment by juneb** [comment]
Ach! Cut/paste error. I'll resubmit.

**Comment by dlwyatt** [comment]
If you add a new commit to your CodeCoverage branch, it'll automatically update the pull request as well.  :)

**Comment by juneb** [comment]
Okay, that's cool. Still learning. Thanks!

**Pull request merged**